### PR TITLE
rename breez/breez-sdk to breez/breez-sdk-greenlight

### PR DESCRIPTION
**Pull request by JssDWt**

### CODE REVIEW

**Review by yaslama — APPROVED** [review]

**Comment by JssDWt** [comment]
@dangeross @ok300 Can you check the remaining 3 commits?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -25,8 +25,8 @@ jobs:
     name: setup
     runs-on: ubuntu-latest
     outputs:
-      sdk-ref: ${{ inputs.sdk-ref || '83740c3595ac16b29896c76a2e8790fba0535ab3' }}
-      package-version: '0.5.0'
+      sdk-ref: ${{ inputs.sdk-ref || '10a4cd62fc39c1e0c56ab2bac3a26210f5515204' }}
+      package-version: '0.5.2'
```
Comment (ok300) [inline .github/workflows/main.yml:29]: Is there a reason why this is not bumped to the latest `0.6.1`?

**Review by ok300 — COMMENTED** [review]
The version (`0.5.0`) is set in a bunch of other files, so the new version has to be updated in them too:
- `src/guide/install.md` L96
- `snippets/dart_snippets/pubspec.lock` L58
- `snippets/kotlin_mpp_lib/shared/build.gradle.kts` L37
- `snippets/swift/BreezSDKExamples/Package.resolved` L8-9
- `snippets/swift/BreezSDKExamples/Package.swift` L11
- `snippets/rust/Cargo.toml` L9 (best to just set the `tag`, if you don't need a custom `rev`)
- `snippets/rust/Cargo.lock` (running `cargo clippy` will update the right lines)

File: .github/workflows/main.yml (hunk 1 above)
Comment (JssDWt) [reply .github/workflows/main.yml:29]: No! It didn't exist yet last week when I created this.

**Review by dangeross — APPROVED** [review]
LGTM

**Pull request merged**